xilem/src/window_options.rs

```diff
@@ -183,5 +193,360 @@ impl<State> WindowOptions<State> {
+        pub(crate) fn build(&self, attrs: WindowAttributes) -> WindowAttributes {
+            let mut attrs = attrs
+                .with_skip_taskbar(self.skip_taskbar)
+                .with_undecorated_shadow(self.decoration_shadow)
+                .with_system_backdrop(self.backdrop_type)
+                .with_border_color(self.border_color)
+                .with_title_background_color(self.title_background_color);
+            if let Some(title_text_color) = self.title_text_color {
+                attrs = attrs.with_title_text_color(title_text_color);
+            }
+            if let Some(corner_preference) = self.corner_preference {
+                attrs = attrs.with_corner_preference(corner_preference);
+            }
+            attrs
+        }
+
+        pub(crate) fn rebuild(&self, prev: &Self, window: &Window) {
+            // TODO: move taskbar_icon to ReactiveWindowAttrs once the winit type implements PartialEq
+            // if self.taskbar_icon != prev.taskbar_icon {
+            //     window.set_taskbar_icon(self.taskbar_icon)
+            // }
+            if self.skip_taskbar != prev.skip_taskbar {
+                window.set_skip_taskbar(self.skip_taskbar);
+            }
+            if self.decoration_shadow != prev.decoration_shadow {
+                window.set_undecorated_shadow(self.decoration_shadow);
+            }
+            if self.backdrop_type != prev.backdrop_type {
+                window.set_system_backdrop(self.backdrop_type);
+            }
+            if self.border_color != prev.border_color {
+                window.set_border_color(self.border_color);
+            }
+            if self.title_background_color != prev.title_background_color {
+                window.set_title_background_color(self.title_background_color);
+            }
+            if self.title_text_color != prev.title_text_color
+                && let Some(c) = self.title_text_color
+            {
+                window.set_title_text_color(c);
+            }
+            if self.corner_preference != prev.corner_preference
+                && let Some(c) = self.corner_preference
+            {
+                window.set_corner_preference(c);
+            }
+        }
+    }
+
+    impl Default for PlatformSpecificInitialWindowAttrs {
+        fn default() -> Self {
+            Self {
+                owner: None,
+                menu: None,
+                taskbar_icon: None,
+                no_redirection_bitmap: false,
+                drag_and_drop: true,
+                class_name: "Window Class".to_string(),
+                clip_children: true,
+            }
+        }
+    }
+
+    /// Extension setters for Windows-specific window options.
+    pub trait WindowOptionsExtWindows {
+        /// Set an owner to the window to be created. Can be used to create a dialog box, for example.
+        // / This only works when [`WindowAttributes::with_parent_window`] isn't called or set to `None`.
```

Comment:
Are these intentionally commented out? Maybe we should put them in a block comment instead if so?

Reply:
Yes, as I intend to follow up with [this](https://github.com/linebender/xilem/pull/1380#issuecomment-3242705139) and then we could uncomment and fix them.